WordPress/Classes/ViewRelated/Post/PostEditor.swift

```diff
@@ -181,9 +198,23 @@ extension PostEditor where Self: UIViewController {
         }
         ContextManager.sharedInstance().saveContextAndWait(context)
     }
+
+    private func postCoordinatorDidUpdate(_ notification: Foundation.Notification) {
+        guard let updatedObjects = (notification.userInfo?[NSUpdatedObjectsKey] as? Set<NSManagedObject>) else {
```

Comment:
The `postCoordinatorDidUpdate` is called frequently during the upload. For example, you can open an unsynced draft post and continue making changes while the previous revision are being uploaded in the background. 

I would suggest adding the previous notification or introducing a new more generic notification for when the latest revision gets destroyed by the coordinator, otherwise the editor will keep creating revisions when it doesn't need to, and it will break the `hasChanges` mechanism.

Reply:
Done! https://github.com/wordpress-mobile/WordPress-iOS/pull/22992/commits/d5dfff314e0c4fe437184121a3292b8836cccc91